README.rst

```diff
@@ -28,16 +29,14 @@ Features
 
 
 Usage
------
-The most straightforward way is to use the :func:`dateparser.parse` function,
-that wraps around most of the functionality in the module.
+=====
 
-.. automodule:: dateparser
-   :members: parse
+The most straightforward way is to use the `dateparser.parse` function,
```

Comment:
@eliasdorneles @Allactaga will it be okay to leave this part? I don't get any red flags for this when I execute the above command: `restview --strict ../README.rst`